.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 2.1
```

Comment:
This is the language spec version. Which is concident, but not the same as the implementation version.  MRI 2.4 supports the language spec for 2.1 (because it's backwards compatible).  This check shows that we aren't using any language constructs newer than 2.1. I belive this is a good thing as people may not be deploying on 2.4.  If we want to start using 2.4 language spec, we should have a discussion about that separately.